packages/@coorpacademy-components/src/molecule/learning-priority-modal/index.js

```diff
@@ -0,0 +1,253 @@
+      cancelButton: {
+        onCancel: handleCancel,
+        label: translate('cancel')
+      },
+      confirmButton: {
+        onConfirm: () => onAdd(selectedPriority),
+        label: translate('add'),
+        iconName: 'plus',
+        disabled: isLoading
+      }
+    };
+  }, [handleCancel, onAdd, translate, selectedPriority, isLoading]);
+
+  if ((!isLoading && !priorities) || !isOpen) return null;
+
+  return (
+    <BaseModal
+      title={translate('learning_priority_modal_title')}
+      description={translate('learning_priority_modal_description')}
+      isOpen={isOpen}
+      onClose={handleClose}
+      footer={footer}
+    >
+      <div className={style.LearningPriorityContainer}>
+        {isLoading ? (
+          <div className={style.loaderContainer}>
+            <Loader className={style.loader} theme="coorpmanager" />
+          </div>
+        ) : (
+          <>
+            <div className={style.searchWrapper}>
+              <SearchForm
+                search={{
+                  placeholder: translate('search_priority_place_holder'),
+                  value: searchValue,
+                  onChange: handleSearch
+                }}
+                onReset={handleSearchReset}
+              />
+            </div>
+            <div className={style.filterWrapper}>
+              {options.length > 2 && searchResults.length > 0
+                ? options.map((filter, index) => {
+                    const {name, value, selected, count} = filter;
+
+                    function handleChange() {
+                      onChange(value);
+                      handleSearchReset();
+                    }
+
+                    if (count === 0) return null;
+
+                    return (
+                      <div key={index} className={style.filterButtonWrapper}>
+                        <FilterButton
+                          active={selected}
+                          filter={name}
+                          onClick={handleChange}
+                          itemTotal={count}
+                        />
+                      </div>
+                    );
+                  })
+                : null}
+            </div>
+            <div className={style.priorityListWrapper}>
+              {priorityList.map(priority => {
+                const {title, ref, courses, type, selected, disabled} = priority;
+                function handlePriorityClick() {
+                  setSelectedPriority(ref);
+                }
+
+                return (
+                  <ListItem
+                    title={title}
+                    subtitle={`${courses} ${translate('courses')}`}
+                    selected={selected || selectedPriority === ref}
+                    disabled={disabled}
+                    onClick={handlePriorityClick}
+                    tags={[
+                      {
+                        label: type,
+                        type: 'default'
+                      }
+                    ]}
```

Comment:
make it compact for readability :clamp:
```suggestion
                    tags={[{label: type,type: 'default'}]}
```